docs/docs/working-with-Etsy-in-Gatsby.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: Working with Etsy in Gatsby
+---
+
+## Etsy in Gatsby
+
+This guide will walk you through the steps needed in order to successfully work with Etsy e-commerce sites in Gatsby.
+
+## Prerequisites
+
+- Create a new [Etsy account](https://www.etsy.com/join) and store if you don’t have one.
+- Create a private app in your store by navigating to Apps, then Manage private apps.
+- Create a new private app, with any “Private app name” and leaving the default permissions as read access under Admin API.
+- Enable the Etsy API by checking the box that says “Allow this app to access your storefront data using Storefront API”.
+- Make sure to also grant access to read product and customer tags by checking their corresponding boxes.
+
+## If you do not already have one ready, [create a Gatsby site](/docs/quick-start):
+
+- From your terminal, you will want to cd or change directory into the folder where you would want your project to live.
+- In your terminal put in the command `npm install -g gatsby -cli` to install the gatsby command-line interface.
+- Next, you would want to create a new project using the command gatsby new [the name of your site].
+- You would then want to cd into your project folder using cd [the name of your site].
+- Run `gatsby develop` to view your project locally.
+
+## [Gatsby-source-etsy ](/packages/gatsby-source-etsy/)
+
+While in your terminal type the command `npm i gatsby-source-etsy`. You will then need to add the plugin to your gatsby-config.js file:
+
+```jsx:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: "gatsby-source-etsy",
+
+      options: {
+        apiKey: "your api key here",
+
+        shopId: "your shop id here",
+
+        language: "en", // optional
+      },
+    },
+  ],
+}
+```
+
+## Etsy API
```

Comment:
I'd expect a Gatsby/Etsy guide would show the user how to pull Etsy listing data into a component, at the very least. Can you add an example showing how this works? The README has some details, they could inspire the guide but should include content written specifically for this learning resource (reference material on how to set up a Gatsby site with an Etsy shop) https://www.gatsbyjs.org/packages/gatsby-source-etsy/